src/isIsogram.test.js

```diff
@@ -6,4 +6,31 @@ describe('isIsogram', () => {
   it(`should be declared`, () => {
     expect(isIsogram).toBeInstanceOf(Function);
   });
+
+  it('should handle empty string', () => {
+    expect(isIsogram('')).toBe(true);
+  });
+
+  it('should handle single character string', () => {
+    expect(isIsogram('a')).toBe(true);
+  });
+
+  it('should handle repeated characters', () => {
+    expect(isIsogram('look')).toBe(false);
+    expect(isIsogram('Adam')).toBe(false);
+  });
+
+  it('should handle non-alphabetic characters', () => {
+    expect(isIsogram('playgrounds!')).toBe(true);
+    expect(isIsogram('123')).toBe(true);
+  });
+
+  it('should handle numbers as input', () => {
+    expect(() => isIsogram(123)).toThrow();
+  });
```

Comment:
This test case expects the `isIsogram` function to throw an error when numbers are passed as input. Ensure that the function implementation includes input validation to handle such cases appropriately.